lib/poll.js

```diff
@@ -4,19 +4,33 @@ export default class Poll extends EE.EventEmitter {
   constructor (options, frequency) {
     super();
     this.url = 'https://' + options.host + '/Remote/GetActiveTableID';
-    this.opts = {
-      shortTableId: options.account + '/' + options.table
-    };
+    this.shortTableId = options.account + '/' + options.table
+
+    if (options.cacheBust) {
+      this.cacheBust = options.cacheBust;
+    } else {
+      this.cacheBust = fasle;
+    }
     this.table = void 0;
     frequency = frequency || 30;
     this.interval = frequency * 1000;
     this.timeout = void 0;
     this.aborted = false;
     this.checkTable();
   }
+  getCacheBust() {
+    // return Math.random().toString().slice(2);
+    return Math.round(Date.now()/1000);
+  }
   checkTable() {
     var self = this;
-    return ajax(this.url, this.opts).then(function (resp) {
+    const opts = {
+      shortTableId: this.shortTableId
+    }
+    if (this.cacheBust) {
+      opts.cacheBust = this.getCacheBust();
```

Comment:
so MapLarge accepts the `cacheBust` property as part of the options object? Is this a secret property that's not mentioned in the documentation?

Reply:
no it could be any key, it was originally just `_` but I made it `cacheBust` for clarity.  The key thing is that the requests are different and thus aren't stored the same in the cache

Reply:
Ahhh, okay. Totally makes sense now.